### PR TITLE
fix: don't hide output from go commands

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

File: pkg/build/golang/docker.go [hunk 1]
```diff
@@ -192,7 +192,7 @@ func (b *DockerGoBuilder) Build(ctx context.Context, in *api.BuildInput, output
 	// Write replace directives.
 	cmd := exec.CommandContext(ctx, "go", append([]string{"mod", "edit"}, replaces...)...)
 	cmd.Dir = plandst
-	_, err = cmd.CombinedOutput()
+	err = cmd.Run()
```
Comment (coryschwartz) [inline pkg/build/golang/docker.go]: This runs successfully, but if the goal is not to hide the command output, the output is still hidden like this.

To display the output you'd have to attach a buffer to stdout and stderr, and then later print out those buffers.

I think the use of CombinedOutput is actually correct, just don't assign the bytes buffer to _.

```suggestion
	output, err := cmd.CombinedOutput()
```

File: pkg/build/golang/docker.go [hunk 2]
```diff
@@ -192,7 +192,7 @@ func (b *DockerGoBuilder) Build(ctx context.Context, in *api.BuildInput, output
...
 	if err != nil {
 		return nil, fmt.Errorf("unable to add replace directives to go.mod; %w", err)
```
Comment (coryschwartz) [inline pkg/build/golang/docker.go]:
```suggestion
		return nil, fmt.Errorf("unable to add replace directives to go.mod; %w", err, output)
```

**Comment by coryschwartz** [comment]
Actually, I was wrong in my previous comments, the err returned by cmd.Run is this one:
https://golang.org/src/os/exec/exec.go?s=13315:13903#L445

The error *does* contain the stderr, but only the process state is returned in ExitError.Error(). If convident the error is always an ExitError, you could print the stderr from there.

File: pkg/build/golang/docker.go (hunk 1 above)
Comment (Stebalien) [reply pkg/build/golang/docker.go]: Ah, you're right. Go sends the output to /dev/null by default.

**Comment by Stebalien** [comment]
@coryschwartz I've changed it to _actually_ send the output to stderr/stdout.

**Pull request merged**